README.md

```diff
@@ -0,0 +1,151 @@
+# Budget app
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [ Video-Demo](#video-demo)
+  - [ Deployment](#deployment)
+  - [ Video Explanation](#video-explanation)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Budget app] <a name="about-project"></a>
+
+The **Budget app** The BudgetApp is a mobile web application designed to help you effectively manage your budget. With this app, you can easily keep track of your transactions and categorize them, allowing you to gain insights into your spending habits. It enables you to monitor how much money you have spent and on what expenses.
+
+## Video Demo <a name="video-demo"></a>
+[video demo](https://drive.google.com/file/d/1WRbTmMTHdQDJpThucKnnXOEF6tEjDoHI/view?usp=sharing)
+
+## Deployment 
+
+[render deploy](https://my-budget-app-i519.onrender.com)
+
```

Comment:
- [ ] Great job working on the requested changes but I think the link is still broken here since after opening it shows this error that the app is not working well so kindly try to fix the problem to have your app deployed well ♻🤩🙏🏻

![image](https://github.com/Surafels/budget-app/assets/58553711/130e15cf-b623-43ce-937b-42053eb63386)